website/content/en/docs/golang/quickstart.md

```diff
@@ -662,7 +662,7 @@ deleted until you remove the finalizer (ie, after your cleanup logic has success
 
 **Example:**
 
-The following is a snippet from the controller file under `pkg/controller/memcached/memcached_controller.go`
+The following is a snippet from the controller file under `test/test-framework/pkg/controller/memcached/memcached_controller.go`
 
```

Comment:
Ditto ^

Reply:
Thanks, @camilamacedo86 , I will change in example after release.